test/functional/bip9-softforks.py

```diff
@@ -200,16 +200,14 @@ def test_BIP(self, bipName, activated_version, invalidate, invalidatePostSignatu
         yield TestInstance([[block, False]])
 
         # Restart all
-        self.test.block_store.close()
+        self.test.clear_all_connections()
         stop_nodes(self.nodes)
-        shutil.rmtree(self.options.tmpdir)
+        shutil.rmtree(self.options.tmpdir + "/node0")
```

Comment:
Not that I know of.

I don't like the fact that this test is using shutil and removing files itself. This change just makes it slightly less damaging (ie it's only removing the datadir rather than the entire tmpdir)